main.py

```diff
@@ -5,7 +5,7 @@
 from discord.ext import commands
 from dotenv import load_dotenv
 
-__version_info__ = ('0', '3', '0')
+__version_info__ = ('0', '6', '1')
```

Comment:
I think I can remove this version number, it's better to track versioning using annotated git tags, this will get forgotten about during updates (And you can't always predict what version it will be before a PR when working within teams)